src/test/java/com/github/stickerifier/stickerify/media/MediaHelperTest.java

```diff
@@ -240,32 +265,40 @@ void noLowFpsAnimatedStickerConversionNeeded() throws Exception {
 	}
 
 	@Test
+	@Tag(Tags.ANIMATED_STICKER)
 	void nonCompliantAnimatedSticker() {
 		var animatedSticker = loadResource("non_compliant_animated_sticker.tgs");
 
-		assertThrows(MediaException.class, () -> MediaHelper.convert(animatedSticker));
+		var ex = assertThrows(MediaException.class, () -> MediaHelper.convert(animatedSticker));
+		assertThat(ex.getMessage(), equalTo("The file with application/gzip MIME type is not supported"));
 	}
 
 	@Test
+	@Tag(Tags.ANIMATED_STICKER)
 	void unsupportedGzipArchive() {
 		var archive = loadResource("unsupported_archive.gz");
 
-		assertThrows(MediaException.class, () -> MediaHelper.convert(archive));
+		var ex = assertThrows(MediaException.class, () -> MediaHelper.convert(archive));
+		assertThat(ex.getMessage(), equalTo("The file with application/gzip MIME type is not supported"));
 	}
 
 	@Test
+	@Tag(Tags.UNSUPPORTED_FILE)
 	void unsupportedFile() {
 		var document = loadResource("document.txt");
 
-		assertThrows(MediaException.class, () -> MediaHelper.convert(document));
+		var ex = assertThrows(MediaException.class, () -> MediaHelper.convert(document));
+		assertThat(ex.getMessage(), equalTo("The file with text/plain MIME type is not supported"));
 	}
 
 	@Nested
+	@Tag(Tags.CONCURRENT)
 	@DisplayName("Concurrently convert")
-	@EnabledIfEnvironmentVariable(named = "CI", matches = "true")
+	@EnabledIfEnvironmentVariable(named = "CI", matches = "(?i)true|1")
```

Comment:
I'd prefer to match only "true", as per GitHub documentation it's the value that variable will have.